crates/bonsaidb-core/src/networking.rs

```diff
@@ -326,6 +332,8 @@ pub enum ServerResponse {
         /// The id of the user created.
         id: u64,
     },
+    /// A user was deleted.
+    UserDeleted,
```

Comment:
Since there is no information being returned, let's remove this variant and have the networking client use `Response::Ok` instead.

Reply:
Done